.devcontainer/devcontainer.json

```diff
@@ -23,6 +23,7 @@
   "customizations": {
     "vscode": {
       "extensions": [
+        "be5invis.toml",
```

Comment:
It looks like it's a TOML extension and doesn't look popular at all ([vscode marketplace link](https://marketplace.visualstudio.com/items?itemName=be5invis.toml), [github link](https://marketplace.visualstudio.com/items?itemName=be5invis.toml)) and isn't maintained, is this one a better idea? https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml ([github link](https://github.com/tamasfe/taplo))

Reply:
That one is not available yet. This PR is a direct port of commits that already exist on `main`.